src/main/java/uk/gov/justice/digital/delius/data/api/ContextlessReferralStartRequest.java

```diff
@@ -17,15 +17,15 @@
 @Builder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
-public class ReferralSentRequest {
+public class ContextlessReferralStartRequest {
 
     @NotNull
     @ApiModelProperty(required = true)
-    private OffsetDateTime sentAt;
+    private OffsetDateTime startedAt;
 
     @NotNull
     @ApiModelProperty(required = true)
-    private UUID serviceCategoryId;
+    private String contractType;
```

Comment:
I wonder if this field could also do with documenting as it's not part of the usual NDelius vocabulary?